src/Frozennode/Administrator/AdministratorServiceProvider.php

```diff
@@ -33,16 +45,9 @@ public function boot()
 		//define a constant that the rest of the package can use to conditionally use pieces of Laravel 4.1.x vs. 4.0.x
 		$this->app['administrator.4.1'] = version_compare(\Illuminate\Foundation\Application::VERSION, '4.1') > -1;
 
-		//set up an alias for the base laravel controller to accommodate >=4.1 and <4.1
-		if (!class_exists('AdministratorBaseController')){ // Verify alias is not already created
-			if ($this->app['administrator.4.1'])
-				class_alias('Illuminate\Routing\Controller', 'AdministratorBaseController');
-			else
-				class_alias('Illuminate\Routing\Controllers\Controller', 'AdministratorBaseController');
-		}
+        class_alias('Illuminate\Routing\Controller', 'AdministratorBaseController');
```

Comment:
The `administrator.4.1` ioc string was just a hack to manage the different locations of the base `Controller` from 4.1 to 4.2. You leave out anything that references this and probably just kill the class alias as well. The controller can just reference `Illuminate\Routing\Controller` now since in L5 this isn't a problem.